src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24878.cs

```diff
@@ -1,5 +1,4 @@
-﻿#if ANDROID || WINDOWS
```

Comment:
The snap shot on windows for this one seems wrong

It still has a white background

I think we should leave FAILS_ON_WINDOWS here and delete the windows snapshot